### ISSUE TITLE
2021JAN Zowe Survey Preparation

### ISSUE THREAD

**Issue by RASakach** [zowe/zac#219]
Complete Survey planning for January 2021

Deadline for questions:  Monday 12/14
Questions should be posted here:  TBD

Guidelines (to-be-considered)
* Limit 5 questions per squad
* Target installation and adoption challenges
* Target decisions that steer 2021 technical direction

**Comment by armstro** [comment]
Hey Rose - getting the survey started sounds like a great idea to avoid being rushed before the call in Jan - couple comments - we can work with John Mertic to use the OMP Survey Monkey account - it worked well before and allows more features to be used in Survey Monkey than a free account (that I used a couple years ago). One thing I used in Survey Monkey was a "grade" on likelihood of getting replies (based on degree of difficulty of the questions) and an estimate of the time a survey will take - I think we want to target for 10-12 mins and not solely rely on the number of questions but the complexity of the questions. 

We need to find a good editor for the questions that squads create to a) avoid duplication b) maybe phrase the questions right to get the best feedback possible. Survey composition is both an art and a science. I do not know who a good editor might be - maybe a designer? who can think about questions without bias and considered the responder? We can discuss who that might be.
